tests/layer_tests/pytorch_tests/test_hstack.py

```diff
@@ -0,0 +1,93 @@
+# Copyright (C) 2018-2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+import torch
+import numpy as np
+import numpy as np
+
+from pytorch_layer_test_class import PytorchLayerTest
+
+class aten_hstack(torch.nn.Module):
+    def forward(self, x):
+        return torch.hstack(self.prepare_input(x))
+    
+    def prepare_input(self, x):
+        return (x, x)
+    
+class aten_hstack_out(aten_hstack):
+    def forward(self, x, out):
+        return torch.hstack(self.prepare_input(x), out=out), out
+    
+class TestHstack(PytorchLayerTest):
+    def _prepare_input(self, out=False, num_repeats=2):
+        data = np.random.randn(2, 1, 3)
+        if not out:
+            return (data, )
+        concat = [data for _ in range(num_repeats)]
+        out = np.zeros_like(np.concatenate(concat, axis=1))
+        return (data, out)
+    
+    @pytest.mark.nightly
+    @pytest.mark.precommit
+    @pytest.mark.parametrize("out", [False, True])
+    def test_hstack(self, out, ie_device, precision, ir_version):
+        model = aten_hstack() if not out else aten_hstack_out()
+        self._test(model, None, "aten_hstack", ie_device, 
+                   precision, ir_version, kwargs={"out": out, "num_repeats": 2})
+        
+    
+class TestHstackAlignTypes(PytorchLayerTest):
+    def _prepare_input(self, in_types):
+        in_vals = []
+        for i in range(len(in_types)):
+            dtype = in_types[i]
+            in_vals.append(np.random.randn(2, 1, 3).astype(dtype))
+        return in_vals
+    
+    def create_model(self, in_count):
+        class aten_align_types_hstack_two_args(torch.nn.Module):
+            def __init__(self):
+                super().__init()
+            
+            def forward(self, x, y):
+                ins = [x, y]
+                return torch.hstack(ins)
+        
+        class aten_align_types_hstack_three_args(torch.nn.Module):
+            def __init__(self):
+                super().__init()
+            
```

Comment:
```suggestion
            def forward(self, x, y):
                ins = [x, y]
                return torch.hstack(ins)

        class aten_align_types_hstack_three_args(torch.nn.Module):
```

AttributeError: 'super' object has no attribute '_aten_align_types_vstack_three_args__init'